en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/102.0a2/whatsnew/
+
+# HTML page title
+developer-mdnplus-page-title = { -brand-name-firefox-developer-edition }
+developer-mdnplus-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+
+# Main title
+developer-mdnplus-more-mdn-your-mdn = More { -brand-name-mdn }. <em>Your</em> { -brand-name-mdn }.
+developer-mdnplus-mdn-is-an-open-source = { -brand-name-mdn-web-docs } is an open-source, collaborative project documenting Web platform technologies, including CSS, HTML, JavaScript and Web APIs. We also provide an extensive set of learning resources for beginning developers and students.
+developer-mdnplus-support-mdn-and-make = Support { -brand-name-mdn } <em>and</em> make it your own.
+
+# CTA button
+developer-mdnplus-get-started = Get Started
+
+developer-mdnplus-whats-included = What’s included
+developer-mdnplus-notifications = Notifications
+developer-mdnplus-development-in-real-time = Development in real time: Get custom alerts
+developer-mdnplus-the-web-doesnt-have-a = The Web doesn’t have a changelog, but { -brand-name-mdn } can help. Follow pages and get customizable notifications when documentation changes, CSS features launch, and APIs ship.
+developer-mdnplus-collections = Collections
+developer-mdnplus-build-your-perfect-library = Build your perfect library. Or let us build it for you.
+developer-mdnplus-no-more-haphazard-hunting = No more haphazard hunting through the vast virtual library: unleash your inner curator and collect your favorite articles in one place for convenient consultation.
+developer-mdnplus-mdn-offline = { -brand-name-mdn } offline
+developer-mdnplus-mdns-entire-library-at-your = { -brand-name-mdn }’s entire library at your fingertips: offline
+developer-mdnplus-taking-your-projects-beyond = Taking your projects beyond the nearest wifi signal? Say goodbye to inaccessible pages or cluttered tabs. With { -brand-name-mdn-plus }, have the fully navigable resources of { -brand-name-mdn } at your disposal even when offline.
```

Comment:
Should `wifi` be `WiFi`?

Reply:
I'm not sure what's official but if we most often use WiFi elsewhere I'd assume that's the standard. I'll update.

Reply:
I looked up else where, the recommendation is Wi-Fi, ^-^